en/mozorg/about/governance/policies/participation.ftl

```diff
@@ -0,0 +1,107 @@
+participation-if-you-feel-you-have = If you feel you have been unfairly accused of violating these guidelines, please follow the same reporting process.
+participation-mozilla-spaces = { -brand-name-mozilla } Spaces
+participation-each-physical-or-virtual = Each physical or virtual { -brand-name-mozilla } space shall have a designated contact.
+participation-mozilla-events = { -brand-name-mozilla } Events
+participation-all-mozilla-events = All { -brand-name-mozilla } events will have designated a specific safety guideline with emergency and anti-abuse contacts at the event as well as online. These contacts will be posted prominently throughout the event, and in print and online materials. Event leaders are requested to speak at the event about the guidelines and to ask participants to review and agree to them when they sign up for the event.
+participation-reports-will-receive = Reports will receive an email notice of receipt. Once an incident has been investigated and a decision has been communicated to the relevant parties, all have the opportunity to appeal this decision by sending an email to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>.
+participation-ask-questions = Ask questions
+participation-everyone-is-encouraged = Everyone is encouraged to ask questions about these guidelines. If you are organizing an event or activity, reach out for tips building inclusion for your event, activity or space. Your input is welcome and you will always get a response within 24 hours (or on the next weekday, if it is the weekend) if you reach out to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>. Please <a href="{ $changelog }">review this change log</a> for updates to this document.
+participation-license-and-attribution = License and attribution
+participation-this-set-of-guidelines = This set of guidelines is distributed under a <a href="{ $license }" rel="license">{ -brand-name-creative-commons } Attribution-ShareAlike license</a>.
+participation-these-guidelines-have = These guidelines have been adapted with modifications from { -brand-name-mozilla }’s original Community Participation Guidelines, the <a href="{ $ubuntu_coc }">Ubuntu Code of Conduct</a>, { -brand-name-mozilla }’s <a href="{ $viewsource_coc }">View Source Conference Code of Conduct</a>, and the <a href="{ $rustlang_coc }">{ -brand-name-rust } Language Code of Conduct</a>, which are based on Stumptown Syndicate’s <a href="{ $citizen_coc }">Citizen Code of Conduct</a>. Additional text from the <a href="{ $lgbtqtech_coc }">LGBTQ in Technology Code of Conduct</a> and the <a href="{ $wiscon_coc }">WisCon code of conduct</a>. This document and all associated processes are only possible with the hard work of many, many Mozillians.
+participation-modifications-to-these = Modifications to these guidelines
```

Comment:
Title case to stay consistent with other titles: `Modifications to these guidelines`